src/pages/experiments/[experimentId]/data-processing/filter-cells/components/CellSizeDistribution/CellSizeDistribution.jsx

```diff
@@ -1,11 +1,18 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
why are these needed?

Reply:
so it doesnt give errors on the <img> elements, because the linter requires them to have events which i dont use